scripts/globals/spells/damage_spell.lua

```diff
@@ -301,6 +301,8 @@ xi.spells.damage.calculateBaseDamage = function(caster, target, spell, spellId,
     -- STEP 4: Spell Damage
     -----------------------------------
     spellDamage = baseSpellDamage + baseSpellDamageBonus + statDiffBonus
+    -- Apply Scarlet Delirium to base damage
+    spellDamage = spellDamage * xi.spells.damage.calculateScarletDeliriumMultiplier(caster)
```

Comment:
This should be handled where all multipliers to base damage are handled.
That is in the very last function of this file.